server/pkg/value/string.go

```diff
@@ -14,6 +14,7 @@ const TypeRichText Type = "richText"
 const TypeMarkdown Type = "markdown"
 const TypeSelect Type = "select"
 const TypeTag Type = "tag"
+const TypeGroup Type = "group"
```

Comment:
GroupID is not a string, it's id.GroupID. Refer to https://github.com/reearth/reearth-cms/blob/group/domain-models/server/pkg/value/reference.go#L12C4-L12C27